scripts/php-mise.sh

```
@@ -0,0 +1,200 @@
+    gpg --keyserver hkps://keys.openpgp.org --recv-keys 0x9D8A98B29B2D5D79 >> "$logs_dir/phive.txt" 2>&1
+
+    if ! gpg --verify "$phive_dir/phive.phar.asc" "$phive_dir/phive.phar" >> "$logs_dir/phive.txt" 2>&1; then
+        echo -e "\e[31mPhive\e[0m failed to verify; skipping install" >&2
+        return 1
+    fi
+
+    {
+        chmod +x "$phive_dir/phive.phar"
+        ln -sf "$phive_dir/phive.phar" "$util_install_bin/phive"
+
+        $phive_dir/phive.phar update-repository-list
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)